pkg/dashboard/static/list-view.js

```diff
@@ -4,6 +4,7 @@ function loadChartsList() {
     const chartsCards = $("#installedList .body")
     chartsCards.empty().append("<div><span class=\"spinner-border spinner-border-sm\" role=\"status\" aria-hidden=\"true\"></span> Loading...</div>")
     $.getJSON("/api/helm/charts").fail(function (xhr) {
+        sendStats('Get charts', {'status': 'failed'});
```

Comment:
It's rather "get releases" here than "charts".  The API call is named by mistake, I plan to address this in #25 
Minor thing, up to you.

Reply:
what do you suggest?